integration-tests/hibernate-orm-graphql-panache/src/main/java/io/quarkus/it/hibertnate/orm/graphql/panache/LocalDateJdbcType.java

```diff
@@ -0,0 +1,104 @@
+package io.quarkus.it.hibertnate.orm.graphql.panache;
+
+import java.io.Serial;
+import java.sql.CallableStatement;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Types;
+import java.time.LocalDate;
+
+import jakarta.persistence.TemporalType;
+
+import org.hibernate.type.descriptor.ValueBinder;
+import org.hibernate.type.descriptor.ValueExtractor;
+import org.hibernate.type.descriptor.WrapperOptions;
+import org.hibernate.type.descriptor.java.JavaType;
+import org.hibernate.type.descriptor.jdbc.BasicBinder;
+import org.hibernate.type.descriptor.jdbc.BasicExtractor;
+import org.hibernate.type.descriptor.jdbc.JdbcLiteralFormatter;
+import org.hibernate.type.descriptor.jdbc.JdbcType;
+import org.hibernate.type.descriptor.jdbc.internal.JdbcLiteralFormatterTemporal;
+import org.hibernate.type.spi.TypeConfiguration;
+
+/**
+ * A "correct" local date JDBC type.
+ */
+public final class LocalDateJdbcType implements JdbcType {
+    @Serial
+    private static final long serialVersionUID = -3375371178728213643L;
+
```

Comment:
This is probably unnecessary, especially since we don't set the serial version UID on any other JdbcType.
Also I hope we never serialize a JdbcType, but... I'm afraid some people probably do :/